lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -204,14 +204,54 @@ def script
 
 if [[ -n "${WRAPPER_EXTENSION}" ]] && [ "`xcrun --find actool`" ] && [ -n "${XCASSET_FILES:-}" ]
 then
-  # Find all other xcassets (this unfortunately includes those of path pods and other targets).
+
+  ASSETS_CAR_PATH="${BUILT_PRODUCTS_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Assets.car"
+  TMP_DIR="$TARGET_TEMP_DIR"
+
+  # TODO: Does PLATFORM_NAME match the input?
+  # TODO: Need to figure out an alternative to parse out `Name` in bash
+  # TODO: Is bash 4 an option? The matching is very inefficient
+  INCLUDED_IMAGESETS=$(xcrun --sdk "$PLATFORM_NAME" assetutil --info "$ASSETS_CAR_PATH" | ruby -e "require 'json'; JSON.parse(STDIN.read).each { |o| puts o['Name'] };")
+  echo "$INCLUDED_IMAGESETS"
+  PODS_XCASSETS="$TMP_DIR/Pods.xcassets"
```

Comment:
where does `$TMP_DIR` match to? Can we use a temp dir provided by Xcode intermediates folder maybe?

Reply:
ah nevermind I see above.